app/models/bishop.rb

```diff
@@ -0,0 +1,4 @@
+class Bishop < Piece
+  scope :bishops, -> {where(type: 'Bishop')}
```

Comment:
@drewsef-916 I like the idea of adding the scope! It could help sometimes.. all the scopes would need to be on the `Piece` model though... so you could say sth like: `Piece.bishops`.. `Bishop.hishops` will get you all the bishops of type bishop :)

Reply:
Thanks, Ilya. That makes sense. I'll make the changes!